app/assets/javascripts/bulk-tagger.js

```diff
@@ -76,6 +77,18 @@
           self.mark_form_as_failed_to_update($(this));
         }
       );
+
+      $element.find(self.selectors.done_form_selector).on(
+        'ajax:success',
+        function() {
+          $(this).parents('.content-item').fadeOut('fast');
```

Comment:
It's not necessary to do this but should we also remove the element from the DOM when the element has faded out?

Reply:
Yeah I was wondering about this. There will be some undesirable interaction between the auto-hide and the bulk-tagger interface too, but I'm not going to address that here. I suspect that we can remove the multi-tag UI now and save the effort of maintaining it.